backend/routes/api.php

```diff
@@ -23,4 +24,7 @@
         return UserResource::make($request->user());
     });
     Route::apiResource('users', UserController::class)->only(['index']);
+    Route::get('/users/{user_id}/accounts', [UserController::class, 'myAccounts']);
```

Comment:
refer to https://laravel.com/docs/8.x/controllers#restful-nested-resources

for nested routing

this should be another apiResource, and function should be index 
ex Route::apiResource('users.accounts', UserAccountController::class)->only('index')

Reply:
pivot may not be the right term haha 

can be `nested` instead